packages/slice-machine/lib/utils/lib.ts

```diff
@@ -1,18 +1,6 @@
 import path from "path";
 import Files from "./files";
 
-enum Prefix {
```

Comment:
What was this used for?

Reply:
For computing which slice lib is local or not. And now this is done by the backend part with the core